pkg/oauth/client.go

```diff
@@ -26,7 +26,7 @@ func NewClient(ctx context.Context, cfg *env.EventMeshConfig) *http.Client {
 	// configure tracing transport
 	client.Transport = &ochttp.Transport{
 		Base:        client.Transport,
-		Propagation: tracecontextb3.TraceContextEgress,
+		Propagation: tracecontextb3.TraceContextEgress(),
```

Comment:
That was a litte hard to understand. How did you catch that? Did it really work before?

Reply:
It was a global variable, I tried to change it to const but it did not work, so I changed it to a function.